src/components/CareCard/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './CareCard.scss';
+
+const CareCard = ({ title, image }) => (
+  <React.Fragment>
```

Comment:
This fragment tag can be avoided, here you only returned one element, this is used when you add multiple elements at the same level.